websites/E/EternalNetwork TM/presence.ts

```diff
@@ -65,7 +65,7 @@ presence.on("UpdateData", async () => {
 			if (pathname.startsWith("/")) presenceData.state = ttl;
 
 			if (pathname.includes("/wp-admin")) {
-				presenceData.state = "Using administrating power over the website !";
+				presenceData.state = "Viewing an admin panel";
 				presenceData.smallImageText = "Admin Panel";
```

Comment:
This isn't needed then

Reply:
Okay, then if I change the message to "Inside Admin Panel" will it make more sense? It's a path of the website where I don't want to show what activity the member does, so that's why I keep this message for all admin panels.

Please, provide me with a message idea of what you think will be good then.

Reply:
Sorry, forgot to click on the exact line. I was trying to say the `smallImageText` here isn't needed as it pretty much repeats `state`

Reply:
Thank you for the fast response, I appreciate it. Can I leave it like this for the admin panels:

```
	presenceData.state = "Viewing the admin panel";
	presenceData.smallImageText = "Administrating";
```

or to leave it empty like `presenceData.smallImageText = "";`, because otherwise will show the URL of the current page.

Reply:
You can leave it blank and use `delete` instead - `delete presenceData.smallImageText`

Reply:
Changes applied.